src/components/plots/abstractXYPlot.ts

```diff
@@ -156,6 +157,14 @@ export module Plot {
       }
     }
 
+    /**
+     * Sets an interpolation mode.
+     */
+    public interpolate(interpolationMode: string) {
+      this._interpolationMode = interpolationMode;
+      return this;
+    }
```

Comment:
Why put this on the abstractXYPlot level?  I don't see the point of interpolation on a ScatterPlot or a BarPlot.